### PR TITLE
Enhancement/skale 2597 Add queues for monitor and report jobs

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: sla_agent.py [hunk 1]
```diff
@@ -154,46 +181,70 @@ def monitor_job(self) -> None:
...
+        monitor_schedule = schedule.every(MONITOR_PERIOD).minutes.do(
+            monitor_w.jobqueue.put, self.monitor_job)
+        report_schedule = schedule.every(REPORT_PERIOD).minutes.do(
+            reporter_w.jobqueue.put, self.report_job)
+        threading.Thread(target=monitor_w.worker, name='Monitor').start()
+        threading.Thread(target=reporter_w.worker, name='Reporter').start()
+        monitor_schedule.run()
+        report_schedule.run()
+
+        # run_threaded(self.monitor_job)
```
Comment (dmytrotkk) [inline sla_agent.py]: Remove comments

File: tools/metrics.py [hunk 2]
```diff
@@ -21,16 +21,28 @@
...
 from web3 import HTTPProvider, Web3
 
+from configs import GOOD_IP, WATCHDOG_PORT, WATCHDOG_URL
+from tools.exceptions import NoInternetConnectionException
+
 logger = logging.getLogger(__name__)
 
 
+def check_internet_connection():
+    if get_ping_node_results(GOOD_IP)['is_offline']:
```
Comment (dmytrotkk) [inline tools/metrics.py]: this cloud be simplified

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: sla_agent.py (hunk 1 above)
Comment (alexgex) [reply sla_agent.py]: done

File: tools/metrics.py (hunk 2 above)
Comment (alexgex) [reply tools/metrics.py]: done

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**